fields-flow/src/commonMain/kotlin/dev/icerock/moko/fields/flow/FormField.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.fields.flow
+
+import dev.icerock.moko.fields.core.FormField
+import dev.icerock.moko.mvvm.flow.CMutableStateFlow
+import dev.icerock.moko.mvvm.flow.CStateFlow
+import dev.icerock.moko.mvvm.flow.cMutableStateFlow
+import dev.icerock.moko.mvvm.flow.cStateFlow
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.flow.stateIn
+
+open class FormField<D, E>(
+    scope: CoroutineScope,
+    initialValue: D,
+    validationTransform: (D) -> E?
```

Comment:
this version not allow us to implement validation of password with confirm password fields.
we should allow validation that use other Flows as inputs (to call `combine`)